ui/app/src/state/reducers/content.ts

```diff
@@ -84,7 +84,7 @@ const updateItemLockState = (state: ContentState, { path, username, locked }) =>
       ...state.itemsByPath,
       [path]: {
         ...state.itemsByPath[path],
-        lockOwner: locked ? username : null,
+        lockOwner: locked ? user : null,
```

Comment:
What is this property now? If it was a string and now an object, sounds like that could causes issues with just these changes alone. 

Also, check typings. We should be getting some form of compilation error.

Reply:
This prop changed from string to a 'Person' object as part of some backend changes, it is already updated in its usages (from `item.lockOwner` to `item.lockOwner?.username`). I'll check to add a proper typing there